pkgs/top-level/aliases.nix

```diff
@@ -1620,6 +1620,7 @@ mapAliases ({
   speech_tools = throw "'speech_tools' has been renamed to/replaced by 'speech-tools'"; # Converted to throw 2022-02-22
   speedometer = throw "speedometer has been removed: abandoned by upstream"; # Added 2022-04-24
   speedtest_cli = throw "'speedtest_cli' has been renamed to/replaced by 'speedtest-cli'"; # Converted to throw 2022-02-22
+  speedtest-exporter = throw "'speedtest-exporter' has been renamed removed as unmaintained"; # Added 2023-07-31
```

Comment:
Please remove "renamed" :-)

Also, please add an entry for prometheus-speedtest-exporter.

Perhaps split in two commits? Or say something about the project being packaged twice?

Reply:
I think each commit should be getting the repo in one direction instead of back and forth. In that model, I'd prefer to squash the alias commits into their respective "pname: remove" commits.